mmdet3d/utils/logger.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from mmcv.utils import get_logger
+
+
+def get_root_logger(log_file=None, log_level=logging.INFO, name='mmdet'):
+    """Get root logger.
```

Comment:
Change the default name to mmdet3d.

Reply:
This is great. I think I could also change the `filter_key` of this logger to be `mmdet3d`, so that the logger information will start with `mmdet3d`.

Reply:
Is it necessary to have a keyword filter for the logger? If not, we can set the logger's name here to `mmdet3d`. However, if the filter is necessary, we have to use consistent name as `train_detector` or `train_segmentor`?

Reply:
Fix.